tcms/static/js/utils.js

```diff
@@ -163,3 +163,34 @@ function treeViewBind() {
       }
     });
 }
+
+function filterTestCasesByProperty(testCases, property, value) {
+    const filterValue = value;
+    const filterBy = property;
+
+    for (let id in testCases) {
+        if (!testCases.hasOwnProperty(id)) {
+            break;
+        }
+
+        let tc = testCases[id];
```

Comment:
is `testCases` a map of `id: testCase`? if so, we can iterate it by doing:
```javascript
Object.entries(testCases).forEach((id, testCase) => {
   ...
})
```
this will make all this checks and assignments redundant.

Reply:
Yes, it is. I have no strong feelings about that.